tools/clang/include/clang/SPIRV/InstBuilder.h

```diff
@@ -147,6 +147,8 @@ class InstBuilder {
                                     spv::StorageClass storage_class);
   InstBuilder &opConstantTrue(uint32_t result_type, uint32_t result_id);
   InstBuilder &opConstantFalse(uint32_t result_type, uint32_t result_id);
+  InstBuilder &opConstant(uint32_t result_type, uint32_t result_id,
```

Comment:
We should put this method declaration at the end of this section since the current surrounding method declarations are all autogen'd from some list.